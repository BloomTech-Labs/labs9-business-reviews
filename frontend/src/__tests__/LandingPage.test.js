import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import LandingPageComponent from '../components/LandingPage';

describe('<LandingPageComponent />', () => {
  it('renders', () => {
    shallow(<LandingPageComponent />);
  });
  it('matches snapshot', () => {
    const wrapper = shallow(<LandingPageComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
  it('should contain LandingNav and Maincontent', () => {
    let land = shallow(<LandingPageComponent />);
    expect(land.find("LandingNav").length).toBe(1);
    expect(land.find("MainContent").length).toBe(1);
  })
});
