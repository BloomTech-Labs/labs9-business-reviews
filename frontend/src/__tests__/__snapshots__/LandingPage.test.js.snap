// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPageComponent /> matches snapshot 1`] = `
<styled.div>
  <LandingNav />
  <MainContent />
</styled.div>
`;
